.env.development

```diff
@@ -6,11 +6,13 @@ REACT_APP_GOOGLE_TAG_SECRET=GTM-K3GR7M5
 REACT_APP_LINK_SSV_DEV_DOCS=https://docs.ssv.network
 REACT_APP_BLOX_API=https://api.stage.bloxstaking.com
 REACT_APP_EXPLORER_URL=https://explorer.stage.ssv.network
+REACT_APP_EXPLORER_VERSION=v2
+REACT_APP_EXPLORER_NETWORK=prater
```

Comment:
If these variables are used only for API calls, name them with `API` in the name: `...EXPLORER_API_...`